npc/other/Global_Functions.txt

```diff
@@ -475,3 +475,17 @@ function	script	F_GetTradeRestriction	{
 		.@trade$ += "NoAuction|";
 	}
 }
+
+//== Function F_MesItemInfo ===================================
+// Show the item name and a clickable link for the item description
+// Only works with mes and mesf, does not work in menu
+function	script	F_MesItemInfo	{
+	.@item = getarg(0);
+	.@itemname$ = getitemname(.@item);
+	if (.@itemname$ == "null")
+		.@itemname$ = "Unknown Item";
+	if (PACKETVER >= 20150729)
```

Comment:
I think for packet version before 2013-01-30 need return item name, because links for this versions not works